### PR TITLE
Features/reviews mobile

### PR DESCRIPTION
**Pull request by PhazZzyo**

### CODE REVIEW

File: src/partials/reviews.html [hunk 1]
```diff
@@ -1 +1,76 @@
-<p>Mandryk</p>
+<section class="reviews">
```
Comment (KhelemeliaOleksii) [inline src/partials/reviews.html]: додати <section class=" section reviews-section">

Comment (KhelemeliaOleksii) [reply src/partials/reviews.html]: `<section class="section reviews-section">`

File: src/partials/reviews.html [hunk 2]
```diff
@@ -1 +1,76 @@
...
+  <div class="container">
```
Comment (KhelemeliaOleksii) [inline src/partials/reviews.html]: замінити ` <div class="container reviews-container">`

File: src/partials/reviews.html (hunk 1 above)
Comment (PhazZzyo) [reply src/partials/reviews.html]: fixed

File: src/partials/reviews.html (hunk 2 above)
Comment (PhazZzyo) [reply src/partials/reviews.html]: fixed

**Review by KhelemeliaOleksii — APPROVED** [review]

**Pull request merged**